kubernetes/test-infra/eks/scripts/delete_orphan_elbs.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash +e
```

Comment:
I like that you're using the errexit flag here. However, the `+e` on this line isn't running on my local system (Fedora 32). It gives me this error:

```
[dakini@dax terraform-provider-kubernetes (eks-cleanup *$%=)]$  kubernetes/test-infra/eks/scripts/delete_orphan_elbs.sh
/usr/bin/env: ‘bash +e’: No such file or directory
/usr/bin/env: use -[v]S to pass options in shebang lines
```

How about something like this instead? You could even add on a pipefail, to catch errors that otherwise would be missed due to ` | ` pipes.

```suggestion
#!/usr/bin/env bash
set -o errexit
set -o pipefail
```

Reply:
Could you also make this file executable? I see our other scripts have execute permissions.

```
[dakini@dax terraform-provider-kubernetes (eks-cleanup *$%=)]$ ls -l scripts/
total 20
-rwxrwxr-x. 1 dakini dakini 294 Apr 30 13:16 build.sh
-rwxrwxr-x. 1 dakini dakini 951 Aug 31 14:06 changelog-links.sh
-rwxrwxr-x. 1 dakini dakini 692 Sep  9 16:14 errcheck.sh
-rwxrwxr-x. 1 dakini dakini 367 Feb 11  2020 gofmtcheck.sh
-rwxrwxr-x. 1 dakini dakini 590 Apr 30 13:16 gogetcookie.sh
```

Reply:
Good call, thanks!